gerrit-entrypoint.sh

```diff
@@ -18,7 +22,7 @@ if [ "$1" = "/gerrit-start.sh" ]; then
 
   if [ -z "$(ls -A "$GERRIT_SITE")" ]; then
     echo "First time initialize gerrit..."
-    gosu ${GERRIT_USER} java -jar "${GERRIT_WAR}" init --batch --no-auto-start -d "${GERRIT_SITE}" ${GERRIT_INIT_ARGS}
+    gosu ${GERRIT_USER} java ${JAVA_OPTIONS} ${JAVA_MEM_OPTIONS} -jar "${GERRIT_WAR}" init --batch --no-auto-start -d "${GERRIT_SITE}" ${GERRIT_INIT_ARGS}
```

Comment:
I see. Actually, container.* options are useless in this image because gerrit.sh isn't utilized to start the service. Those JAVA_* variables was introduced by a patch with a lot of variables. I totally forget the existence of them. ：P
Anyway, your work makes those JAVA_* variables useful, finally. That's great.